Sources/FoundationInternationalization/Formatting/Date/Date+IntervalFormatStyle.swift

```diff
@@ -54,29 +54,13 @@ extension Date {
 
         public func format(_ v: Range<Date>) -> String {
             let formatter = Self.cache.formatter(for: self) {
-                let option: ICUPatternGenerator.HourCycleOption?
-                if locale.force24Hour {
-                    option = .force24Hour
-                } else if locale.force12Hour {
-                    option = .force12Hour
-                } else {
-                    option = nil
-                }
-
-                var template: String
-                if let option {
-                    let pattern = ICUPatternGenerator.localizedPatternForSkeleton(localeIdentifier: locale.identifier, calendarIdentifier: calendar.identifier, skeleton: symbols.formatterTemplate, hourCycleOption: option)
```

Comment:
We used to need to pass the template to `ICUPatternGenerator` to resolve 12/24-hour preferences override in Locale because of a bug in ICU's date interval formatter. The bug was fixed a while ago so this pass-through is no longer needed.